sdk/storage/azure-storage-blobs/cgmanifest.json

```diff
@@ -10,16 +10,6 @@
       },
       "DevelopmentDependency": true
     },
-    {
-      "Component": {
-        "Type": "git",
-        "git": {
-          "RepositoryUrl": "https://github.com/nlohmann/json",
-          "CommitHash": "db78ac1d7716f56fc9f1b030b715f872f93964e4"
-        }
-      },
-      "DevelopmentDependency": true
```

Comment:
This should have been false, rather than true, since the 3rd party JSON library isn't just a dev dependency.